### ISSUE TITLE
Update notification persists

### ISSUE THREAD

**Issue by ubercam** [burst-apps-team/Qbundle#6]
The red label next to the current market cap which says (New update available) is constantly showing, despite being 100% up to date. It's been doing this for several versions.

<img width="426" alt="2019-09-23 23_18_50-Application manager" src="https://user-images.githubusercontent.com/7612333/65467768-27f06680-de5a-11e9-918f-13efb0833108.png">

Restarting the wallet or PC has nothing but a temporary effect at best. It might not show up instantly, but it will.

Thanks

**Comment by harryjph** [comment]
Yep, the update notification is somewhat completely broken, I will see if I can try and fix this.
